src/ui/server/helpers/is-policy-contact-data-same-as-owner/index.test.ts

```diff
@@ -0,0 +1,43 @@
+import isPolicyContactDataSameAsOwner from '.';
+import { mockContact, mockApplication } from '../../test-mocks';
+
+const { owner } = mockApplication;
+
+describe('server/helpers/is-policy-contact-data-same-as-owner', () => {
+  describe('when policyContact and owner are the different', () => {
+    it('should return false', () => {
+      const result = isPolicyContactDataSameAsOwner(owner, mockContact);
+
+      expect(result).toEqual(false);
+    });
+  });
+
+  describe('when policyContact and owner are the same', () => {
+    const sameOwner = {
+      ...owner,
+      ...mockContact,
+    };
+
+    it('should return true', () => {
+      const result = isPolicyContactDataSameAsOwner(sameOwner, mockContact);
+
+      expect(result).toEqual(true);
+    });
+  });
+
```

Comment:
Can we add more extensive coverage for:
- All matches, only the first name is different
- All matches, only the last name is different
- All matches, only the email is different

Or have a look at the test coverage reporter for this file I suspect some lines are not covered